packages/calendar/test/calendar.test.ts

```diff
@@ -10,12 +10,24 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+import sinon, { spy } from 'sinon';
```

Comment:
SWC doesn't use sinon. Can you use testing helpers from [here](https://github.com/adobe/spectrum-web-components/blob/mirekszot/date-time-picker-ui-fixes/test/testing-helpers.ts#L0-L1)